core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/docker.yml

```diff
@@ -41,12 +43,12 @@
     - name: Remove docker-ce and docker-ce-selinux
       block:
         - name: Remove docker-ce
-          package:
+          yum:
             name: docker-ce-{{ ansible_facts.packages['docker-ce'][0].version }}
             state: absent
 
         - name: Remove docker-ce-selinux
-          package:
+          yum:
             name: docker-ce-selinux-{{ ansible_facts.packages['docker-ce-selinux'][0].version }}
             state: absent
           when: "'docker-ce-selinux' in ansible_facts.packages"
```

Comment:
Do we need the last condition? If no, can it be combined to the one task?

Reply:
I don't really know at this point :thinking: @to-bar can you shed some light here? :+1: :innocent:

Reply:
Today I would write this as follow:

```
- name: Remove Docker in version < 18.09 (RedHat)
  when: ansible_os_family == "RedHat"
  module_defaults:
    yum: { lock_timeout: 300 }
  block:
    - name: Get information about installed packages as facts
      package_facts:
        manager: auto

    - name: Remove docker-ce and docker-ce-selinux
      yum:
        name:
          - docker-ce
          - docker-ce-selinux
        state: absent
      when:
        - "'docker-ce' in ansible_facts.packages"
        - ansible_facts.packages['docker-ce'][0].version is version ('18.09', '<')
```
but not tested :confused:

Reply:
I think another PR would be a better place for this :)